lang/csharp/src/apache/main/IO/Resolver.cs

```diff
@@ -15,14 +15,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 using System;
 using System.Collections.Generic;
 using Newtonsoft.Json.Linq;
-using Newtonsoft.Json;
 
 namespace Avro.IO
 {
-    static class Resolver
+    internal static class Resolver
```

Comment:
non-functional ?

Reply:
Although we may need to revisit what is publicly exposed.  Things that now have access modifiers were not accessible before since the compiler will automatically make them internal or private.